share/spice/spotify/spotify.js

```diff
@@ -0,0 +1,38 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_spotify = function(api_result){
+
+        var query = DDG.get_query();
+        query = query.replace(/spotify/, ""); //replace trigger words from query
+
+        // Validate the response (customize for your Spice)
+        if (!api_result || api_result.tracks.items.length === 0) {
```

Comment:
When requesting tracks, the API always returns an object with a `tracks` object with an `items` array, even if the query doesn't match anything. Eg https://api.spotify.com/v1/search?q=sadasodsjioadsjidosajdisoa&limit=35&type=track

So I thought it wasn't going to add any extra safety when checking the response.